scalardb-test/src/main/java/kelpie/scalardb/Common.java

```diff
@@ -101,7 +101,8 @@ public static DatabaseConfig getDatabaseConfig(Config config) {
     props.setProperty(DatabaseConfig.NAMESPACE_PREFIX, prefix);
     props.setProperty(DatabaseConfig.TRANSACTION_MANAGER, transactionManager);
     props.setProperty(DatabaseConfig.ISOLATION_LEVEL, isolationLevel);
-    props.setProperty(DatabaseConfig.SERIALIZABLE_STRATEGY, serializableStrategy);
+//    props.setProperty(ConsensusCommitConfig.SERIALIZABLE_STRATEGY, serializableStrategy);
```

Comment:
We can't use this one?

Reply:
Thank you for reviewing this!

> We can't use this one?

Yes for now. `ConsensusCommitConfig` will be introduced in the next release (3.2.0). So I will use it after 3.2.0 is released.